pkg/webservice/routes.go

```diff
@@ -102,7 +102,15 @@ var webRoutes = routes{
 		"Scheduler",
 		"PUT",
 		"/ws/v1/config",
-		updateConfig,
+		updateClusterConfig,
+	},
+
+	// endpoint to update the current conf
```

Comment:
please update the comment here, since this is the endpoint for validating the config, not for updating it.

Reply:
Taken care